lost-paws/lost-paws
# Registration form with UI, cookies, and routing

## Overview

**Issue Type**


**Description**
This feature will enable users to route to localhost:3000/register from the home page by selecting the register button.
They will also be able to create an account using the input fields which then send data to our database
This PR also enables cookie validation with their unique_id 
This PR also has styling components for the registration form

**Ticket Item**
This ticket covers 4 different tickets labeled in light green in trello.

**Steps to Validate Feature **

1. Selecting the button `register` from the front end should navigate to localhost:3000/register
2. Adding data in the input fields should send a request to our database
3. Selecting `register` should succeed and re-route the user back to the home page
4. Opening the console.logs should provide your unique cookie
5. There should be styling present in the UI
